src/Compilers/VisualBasic/Portable/Binding/Binder_Expressions.vb

```diff
@@ -341,6 +341,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     If Not CheckTupleMemberName(name, i, argumentSyntax.NameColonEquals.Name, diagnostics, uniqueFieldNames) Then
                         hasErrors = True
                     End If
+
+                    '  check type character
+                    Dim typeChar As TypeCharacter = nameSyntax.Identifier.GetTypeCharacter()
+                    If typeChar <> TypeCharacter.None Then
+                        ReportDiagnostic(diagnostics, nameSyntax, ERRID.ERR_TupleLiteralDisallowsTypeChar)
+                        hasErrors = True
```

Comment:
fixed now

---
In reply to: [82277984](https://github.com/dotnet/roslyn/pull/14284#discussion_r82277984) [](ancestors = 82277984,82057535)